pynucastro/networks/tests/test_screening.py

```diff
@@ -37,7 +37,7 @@ def test_screening_chugunov_2007(self, rc):
 
         rates = {"C12 + He4 --> O16 <nac2_reaclib__>": 5.794539791829924,
                  "C12 + C12 --> He4 + Ne20 <cf88_reaclib__>": 103.21274049093526,
-                 "C12 + C12 --> n + Mg23 <cf88_reaclib__reverse>": 103.21274049093526,
+                 "C12 + C12 --> n + Mg23 <cf88_reaclib__derived_from_inverse>": 103.21274049093526,
```

Comment:
Right, Q<0 for this reaction which makes it inverse, and is "derived from inverse" to compute the forward reaction $Q>0$.